config/docs-json.config.js

```diff
@@ -46,7 +47,52 @@ StyleDictionary.registerFormat({
  *   comment: '...'
  * },
  */
-function getDocsShape() {
+
+ var levels = {
+	"fail": {
+		range: [0, 3],
+		color: "hsl(0, 100%, 40%)"
+	},
+	"aa-large": {
+		range: [3, 4.5],
+		color: "hsl(40, 100%, 45%)"
+	},
+	"aa": {
+		range: [4.5, 7],
+		color: "hsl(80, 60%, 45%)"
+	},
+	"aaa": {
+		range: [7, 22],
+		color: "hsl(95, 60%, 41%)"
+	}
+};
+
+function getLevel(ratio) {
+  for (const [key, value] of Object.entries(levels)) {
+    if (ratio > value.range[0] && ratio < value.range[1]) {
+      return key
+    }
+  }  
+}
+
+function getContrastCheck(value, element, allTokens) {
+  const format = element.replaceAll('.', '-').replace('&', 'and')
+  const withPrefix = 'telekom-' + format
+  const currentToken = allTokens.find(el => el.name === withPrefix)
+  let contrastRatio;
+  if (currentToken && currentToken.value) {
+    contrastRatio = hex(value, currentToken.value)
+  }
+  return {
+    "path": element,
+    "name": humanCase(element),
```

Comment:
This name needs to be like in the main array… (e.g. `"name": "Text & Icon / Standard"`)

This is the relevant bit https://github.com/telekom/design-tokens/blob/main/config/docs-json.config.js#L56

Reply:
added this, but it may be related to the other open comment https://github.com/telekom/design-tokens/pull/143#discussion_r804450795 as I am manipulating the string in the function directly...if we change the token names/structure then that will stop working